src/applications/pensions/config/form.js

```diff
@@ -569,6 +576,23 @@ const formConfig = {
                     'ui:widget': 'radio',
                     'ui:required': (...args) => !isCurrentMarriage(...args),
                   },
+                  separationOtherExplanation: {
```

Comment:
this should be called `otherExplanation` instead of `separationOtherExplanation `, for consistency's sake. Otherwise, this will cause schema validation failures. Please update all references, including the test fixtures.